### PR TITLE
Add MMS endpoints and Mintscan explorer for Teritori

### PR DESCRIPTION
**Pull request by bilalkhan360**

### CODE REVIEW

File: teritori/chain.json [hunk 1]
```diff
@@ -95,7 +95,11 @@
       {
         "address": "https://teritori-rpc.ibs.team",
         "provider": "Inter Blockchain Services"
-      }
+      },
+      {
+        "address": "https://rpc-teritori.mms.team/",
+        "provider": "MMS"
+      },
```
Comment (JeremyParish69) [inline teritori/chain.json:102]: remove comma

File: teritori/chain.json [hunk 2]
```diff
@@ -129,8 +133,12 @@
       {
         "address": "https://teritori-api.ibs.team",
         "provider": "Inter Blockchain Services"
-      }
-    ],
+      },
+      {
+        "address": "https://api-teritori.mms.team/",
+        "provider": "MMS"
+      },
```
Comment (JeremyParish69) [inline teritori/chain.json:140]: remove comma

File: teritori/chain.json [hunk 3]
```diff
@@ -155,9 +163,13 @@
...
         "address": "https://teritori-grpc.brocha.in",
         "provider": "Brochain"
-      }
-    ]
-  },
+      },
+      {
+        "address": "https://grpc-teritori.mms.team/"
+        "provider" : "MMS"
+      },
```
Comment (JeremyParish69) [inline teritori/chain.json:170]: remove comma

File: teritori/chain.json [hunk 4]
```diff
@@ -184,5 +196,10 @@
       "url": "https://explorer.brocha.in/teritori",
       "tx_page": "https://explorer.brocha.in/teritori/tx/${txHash}"
     }
+    {
+      "kind": "mintscan",
+      "url": "https://www.mintscan.io/teritori",
+      "tx_page": "https://www.mintscan.io/teritori/txs/${txHash}"
+    },
```
Comment (JeremyParish69) [inline teritori/chain.json:203]: remove comma

File: teritori/chain.json [hunk 5]
```diff
@@ -184,5 +196,10 @@
       "url": "https://explorer.brocha.in/teritori",
       "tx_page": "https://explorer.brocha.in/teritori/tx/${txHash}"
     }
```
Comment (JeremyParish69) [inline teritori/chain.json:198]: add comma please

**Comment by lukfm-mms** [comment]
thanks for contributing our endpoints=)

File: teritori/chain.json [hunk 6]
```diff
@@ -157,12 +165,20 @@
...
+        "address": "https://grpc-teritori.mms.team/"
+        "provider" : "MMS"
+      },
       {
         "address": "teritori-grpc.polkachu.com:15990",
         "provider": "Polkachu"
       }
-    ]
-  },
+    ],
```
Comment (JeremyParish69) [inline teritori/chain.json:180]: sorry, can't have this comma here

File: teritori/chain.json [hunk 7]
```diff
@@ -134,8 +138,12 @@
       {
         "address": "https://teritori-api.ibs.team",
         "provider": "Inter Blockchain Services"
+      },
+      {
+        "address": "https://api-teritori.mms.team/",
+        "provider": "MMS"
       }
-    ],
+        ],
```
Comment (JeremyParish69) [inline teritori/chain.json:146]: sorry, can't have this comma here

File: teritori/chain.json [hunk 8]
```diff
@@ -157,12 +165,20 @@
...
+        "address": "https://grpc-teritori.mms.team/"
+        "provider" : "MMS"
+      },
       {
         "address": "teritori-grpc.polkachu.com:15990",
         "provider": "Polkachu"
       }
     ]
-  },
+  }
```
Comment (JeremyParish69) [inline teritori/chain.json:181]: we also need a comma here, because there is an object after it

File: teritori/chain.json [hunk 9]
```diff
@@ -157,6 +165,14 @@
         "address": "https://teritori.nodejumper.io:9090",
         "provider": "Nodejumper 🚀"
       },
+      {
+        "address": "https://teritori-grpc.brocha.in",
+        "provider": "Brochain"
+      },
+      {
+        "address": "https://grpc-teritori.mms.team/"
```
Comment (JeremyParish69) [inline teritori/chain.json:173]: missing comma here, too

**Pull request merged**